### PR TITLE
Moved VCR config to spec_helper, enabled rake command, added documentation to gitignore

### PR DESCRIPTION
**Pull request by lmcglohon**
Removed ruby 1.9.3

@cbeer

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ script: rake rspec
 rvm:
   - 2.1.4
```
Comment (cbeer) [inline .travis.yml]: We should validate this list of ruby versions against what's being used in production environments. Ideally, we'd test with ruby 2.2, and/or upgrade applications using Ruby 2.0 to something newer.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,21 +1,3 @@
...
-# individual file that may not need all of that loaded. Instead, consider making
-# a separate helper file that requires the additional dependencies and performs
-# the additional setup, and require it from the spec files that actually need
-# it.
-#
-# The `.rspec` file also contains a few flags that are not defaults but that
-# users commonly want.
-#
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require 'bundler/setup'
```
Comment (cbeer) [inline spec/spec_helper.rb]: I wonder if these lines were intended to support Ruby 1.9, or something? I don't think I've ever had to do them in a gem before..

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -24,80 +6,24 @@
...
-  #   # the seed, which is printed after each run.
-  #   #     --seed 1234
-  #   config.order = :random
-  #
-  #   # Seed global randomization in this process using the `--seed` CLI option.
-  #   # Setting this allows you to use `--seed` to deterministically reproduce
-  #   # test failures related to randomization by passing the same `--seed` value
-  #   # as the one that triggered the failure.
-  #   Kernel.srand config.seed
+VCR.configure do |config|
```
Comment (cbeer) [inline spec/spec_helper.rb]: Could we move this block after the `require 'vcr'` line?

**Comment by lmcglohon** [comment]
@cbeer - I think it is ready based upon your comments

**Comment by cbeer** [comment]
:+1:

**Pull request merged**